### PR TITLE
[HOLD] Added schema property to skyuxconfig.json

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Depends on https://github.com/blackbaud/skyux-builder/pull/347 being released first.

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Closing this in favoring of adding a PR to https://github.com/schemastore/schemastore/

**Comment by Blackbaud-BobbyEarl** [comment]
After more carefully reading https://code.visualstudio.com/docs/languages/json, opening this PR back up.  Also note, this is how `angular-cli.json` refers to its schema.  https://github.com/angular/angular-cli/blob/d21caada668f2ff0255b0353b735d9966d17ee31/tests/e2e/assets/1.0.0-proj/.angular-cli.json

File: skyuxconfig.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "mode": "easy",
   "compileMode": "aot",
-  "a11y": true
+  "a11y": true,
+  "$schema": "./node_modules/@blackbaud/skyux-builder/skyuxconfig-schema.json"
```
Comment (Blackbaud-SteveBrush) [inline skyuxconfig.json:5]: We'll need to change this to `"$schema": "./node_modules/@skyux/config/skyuxconfig-schema.json",` once `@skyux/config` is released.

Comment (Blackbaud-BobbyEarl) [reply skyuxconfig.json:5]: Good point.  Perhaps it's best to just hold off until the v3 migration has started to add this property.

**Comment by Blackbaud-SteveBrush** [comment]
Closing in favor of https://github.com/blackbaud/skyux-template/pull/60

**Pull request closed without merging**